lib/src/base-styles/_nx-tile.scss

```diff
@@ -157,8 +157,21 @@ $nx-tile-background-color: $nx-white;
 
```

Comment:
Should this use container-vertical instead of container-horizontal?  That would allow you to get rid of the `margin: 0` properties below

Reply:
> That would allow you to get rid of the margin: 0 properties below

And probably that sibling selector part too

Reply:
> Should this use container-vertical instead of container-horizontal? That would allow you to get rid of the `margin: 0` properties below

https://github.com/sonatype/sonatype-react-shared-components/pull/124/commits/6bc60547bcdc20d4f8eb364a2e584b0f091870e1

Reply:
> > That would allow you to get rid of the margin: 0 properties below
> 
> And probably that sibling selector part too

The 24px top margin is still custom

Reply:
> The 24px top margin is still custom

Currently, in general NxAlert has 8px vertical margins, and nx-btn-bar has 12px vertical margins.  So am I to understand that in a footer the space between these two elements is non-standard? Or is one (or both) of those two margins I mentioned actually supposed to be 24px?

If we really do want this margin to be non-standard, it could probably be implemented more simply as a `margin-bottom` on `.nx-tile-footer .nx-alert` rather than using a sibling selector.

Reply:
> If we really do want this margin to be non-standard, it could probably be implemented more simply as a `margin-bottom` on `.nx-tile-footer .nx-alert` rather than using a sibling selector.

I thought of that, but then the bottom margin is always on the `.nx-alert` so if there isn't any object below it then as it stands right now there's be 24px of bottom margin on the alert and 24px of padding on the footer which is 48px of space.

Reply:
> I thought of that, but then the bottom margin is always on the .nx-alert so if there isn't any object below it then as it stands right now there's be 24px of bottom margin on the alert and 24px of padding on the footer which is 48px of space.

Again, this is what `container-vertical` is for.  In general every block component should declare the minimum distance it wants to be from siblings, and its up to the container to strip off that margin from its first and last children.

Reply:
> > I thought of that, but then the bottom margin is always on the .nx-alert so if there isn't any object below it then as it stands right now there's be 24px of bottom margin on the alert and 24px of padding on the footer which is 48px of space.
> 
> Again, this is what `container-vertical` is for. In general every block component should declare the minimum distance it wants to be from siblings, and its up to the container to strip off that margin from its first and last children.

Right: https://github.com/sonatype/sonatype-react-shared-components/pull/124/commits/b2f1df5bd9c05c1dcd6e31676a065f9910e6734c